scripts/tokenomics-emulation/generate-chain-spec.sh

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 
-rm -rf spec-data/customSpec.json
-rm -rf spec-data/customSpecRaw.json
+docker-compose down -t 0
```

Comment:
Why `0`?

Reply:
It's removes old containers in 10 times faster

Reply:
Ok, here we don't have any post-termination actions. So it is safe.